docs/src/reference/mount.md

```diff
@@ -266,6 +271,36 @@ When running in write mode, the lakeFS URI must point to a branch, not a commit
 
 ---
 
+## Everest for Windows (Cloud Filter API)
```

Comment:
We need some section about behavior such as cache management: 
- Files are first stored as stubs "placeholders" not having actual content only metadata
- When files are accessed they get "hydrated" their content is stored on device, Subsequent reads serving data directly from local cache without reaching to lakeFS api.
- Currently, any access to the file content (i.e read first byte) will force the full download of the file.
- Eviction policy: The OS will dehydrate (clear content) under storage pressure and After un-mount all the data get's deleted from local device. 
- Current limitation Windows File Explorer UI: might work very slow in many setups it performs recursive indexing of the whole directory.